test/test_quantization.py

```diff
@@ -290,6 +290,72 @@ def checkQuantized(model):
         checkQuantized(model)
 
 
+class ScriptabilityTest(QuantizationTestCase):
+    def test_wrapper_traceable(self):
+        UnaryWrapper = nnq.modules.wrapper_module.UnaryWrapper
+        QUnaryWrapper = nnq.modules.wrapper_module.QuantizedUnaryWrapper
+        script_module = torch.jit.trace(UnaryWrapper(torch.cat),
+                                        (torch.tensor(0), torch.tensor(0)))
+        script_qmodule = torch.jit.trace(QUnaryWrapper(torch.cat),
+                                         (torch.tensor(0), torch.tensor(0)))
+
+        a = torch.tensor(24)
+        b = torch.tensor(42)
+        y = torch.cat((a, b))
+        y_hat = script_module((a, b))
+        self.assertEqual(y, y_hat,
+                         "Results of the traced unary wrapper are off!")
+        y_hat = script_qmodule((a, b))
+        self.assertEqual(y, y_hat,
+                         "Results of the traced unary qwrapper are off!")
+
+        BinaryWrapper = nnq.modules.wrapper_module.BinaryWrapper
+        QBinaryWrapper = nnq.modules.wrapper_module.QuantizedBinaryWrapper
+        script_module = torch.jit.trace(BinaryWrapper(torch.add),
```

Comment:
Need to have check_trace=False for tracing to work. I have filed a github issue () to track this bug.
https://github.com/pytorch/pytorch/issues/23993